public/index.html

```diff
@@ -25,16 +25,27 @@ <h1 aria-label="app text">
           thinking for you.
         </h1>
         <section aria-label="allergies section" class="allergies">
-          <input type="checkbox" class="lactose-intolerant"/>
+          <input type="checkbox" class="lactose-intolerant" />
           <div>Are you lactose intolerant?</div>
+          <input type="checkbox" class="vegan" />
+          <div>Are you Vegan?</div>
         </section>
-        <section aria-label="cusine-options" class="cusine-options">
+        <!-- <section aria-label="cusine-options" class="cusine-options">
```

Comment:
Commented-out `HTML` or code can be removed, which makes code reviews easier. :-)